### PR TITLE
[zh] sync docs/reference/config-api/apiserver-config.v1beta1.md

### PR DESCRIPTION
**Pull request by JarHMJ**
[zh] sync docs/reference/config-api/apiserver-config.v1beta1.md

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
See preview: https://deploy-preview-41172--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/config-api/apiserver-config.v1beta1/

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 1]
```diff
@@ -0,0 +1,516 @@
...
+
+
+<!-- 
+## Resource Types  
+-->
+## 资源类型   {#resource-types}
+
+- [EgressSelectorConfiguration](#apiserver-k8s-io-v1beta1-EgressSelectorConfiguration)
+- [TracingConfiguration](#apiserver-k8s-io-v1beta1-TracingConfiguration)
+
```
Comment (my-git9) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:27]: Just change it to one blank line, and the same goes for other places

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+title: kube-apiserver Configuration (v1beta1)
+content_type: tool-reference
+package: apiserver.k8s.io/v1beta1
+auto_generated: true
+-->
+
+<!-- 
+Package v1beta1 is the v1beta1 version of the API.</p> 
+-->
+<p>包 v1beta1 包含 API 的 v1beta1 版本。</p>
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:16]:
```suggestion
<p>v1beta1 包是 v1beta1 版本的 API。</p>
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+<code>string</code>
+</td>
+<td>
+   <p>
+   <!-- name is the name of the egress selection.
+   Currently supported values are &quot;controlplane&quot;, &quot;master&quot;, &quot;etcd&quot; and &quot;cluster&quot;
+   The &quot;master&quot; egress selector is deprecated in favor of &quot;controlplane&quot; -->
+  name 是 Egress 选择器的名称。当前支持的取值有 &quot;controlplane&quot;，
+  &quot;master&quot;，&quot;etcd&quot; 和 &quot;cluster&quot;。
+  &quot;master&quot; Egress 选择器已被弃用，推荐使用 &quot;controlplane&quot;。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:165]:
```suggestion
   <!--
   name is the name of the egress selection.
   Currently supported values are &quot;controlplane&quot;, &quot;master&quot;, &quot;etcd&quot; and &quot;cluster&quot;
   The &quot;master&quot; egress selector is deprecated in favor of &quot;controlplane&quot;
   -->
   name 是 Egress 选择算符的名称。当前支持的取值有 &quot;controlplane&quot;、
   &quot;master&quot;、&quot;etcd&quot; 和 &quot;cluster&quot;。
   &quot;master&quot; Egress 选择算符已被弃用，推荐使用 &quot;controlplane&quot;。
```
参考 [en-zh glossary](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/)，一个页面上可以把`选择器`和`选择算符`统一一下。

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 4]
```diff
@@ -0,0 +1,470 @@
...
+<code>string</code>
+</td>
+<td>
+   <p>
+   <!--
+   caBundle is the file location of the CA to be used to determine trust with the konnectivity server.
+   Must be absent/empty if TCPTransport.URL is prefixed with http://
+   If absent while TCPTransport.URL is prefixed with https://, default to system trust roots.
+   -->
+   caBundle 是指向用来确定与 konnectivity 服务器间信任欢喜的 CA 证书包的文件位置。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:283]:
```suggestion
   caBundle 是指向用来确定与 konnectivity 服务器间信任关系的 CA 证书包的文件位置。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 5]
```diff
@@ -0,0 +1,470 @@
...
+
+<tr><td><code>proxyProtocol</code> <B><!--[Required]-->[必需]</B><br/>
+<a href="#apiserver-k8s-io-v1beta1-ProtocolType"><code>ProtocolType</code></a>
+</td>
+<td>
+   <p>
+   <!-- 
+   Protocol is the protocol used to connect from client to the konnectivity server. 
+   -->
+   Protocol 是客户端连接到 konnectivity 服务器所使用的协议。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:114]:
```suggestion
   proxyProtocol 是客户端连接到 konnectivity 服务器所使用的协议。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 6]
```diff
@@ -0,0 +1,470 @@
...
+</tr>
+<tr><td><code>connection</code> <B><!--[Required]-->[必需]</B><br/>
+<a href="#apiserver-k8s-io-v1beta1-Connection"><code>Connection</code></a>
+</td>
+<td>
+   <p>
+   <!--
+   connection is the exact information used to configure the egress selection
+   -->
+   connection 是用来配置 Egress 选择器的配置信息。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:177]:
```suggestion
   connection 是用来配置 Egress 选择算符的配置信息。
```

File: content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md [hunk 7]
```diff
@@ -0,0 +1,470 @@
...
+<code>int32</code>
+</td>
+<td>
+   <p>
+   <!--
+   SamplingRatePerMillion is the number of samples to collect per million spans.
+   Recommended is unset. If unset, sampler respects its parent span's sampling
+   rate, but otherwise never samples.
+   -->
+   SamplingRatePerMillion 是每百万 span 中采集的样本数。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/config-api/apiserver-config.v1beta1.md:464]:
```suggestion
   samplingRatePerMillion 是每百万 span 中采集的样本数。
```

**Pull request merged**